pkg/utils/etcdutil/etcdutil.go

```diff
@@ -433,12 +433,16 @@ func CreateClients(tlsConfig *tls.Config, acUrls []url.URL) (*clientv3.Client, *
 
 // createHTTPClient creates a http client with the given tls config.
 func createHTTPClient(tlsConfig *tls.Config) *http.Client {
-	return &http.Client{
-		Transport: &http.Transport{
-			DisableKeepAlives: true,
-			TLSClientConfig:   tlsConfig,
-		},
-	}
+	// FIXME: Currently, there is no timeout set for certain requests, such as GetRegions,
```

Comment:
DisableKeepalive is in many releases, do we need to pick this pr for the older versions?

Reply:
I think there may be some reason to open `DisableKeepalive` in the old version, let's only pick the recent version first.